rbi/core/thread.rbi

```diff
@@ -1118,8 +1139,8 @@ class Thread < Object
   # ...
   # flag = false # stop thread
   # ```
-  sig {params(args: T.untyped).returns(T::Boolean)}
-  def self.pending_interrupt?(*args); end
+  sig {params(error: T::Class[T.anything]).returns(T::Boolean)}
+  def self.pending_interrupt?(error); end
```

Comment:
Should `error` be optional? It seems like that might be the case from the documentation comment but I have not checked this against the upstream docs.

Reply:
Yes, that's my bad - will fix.